crates/workspace/src/workspace.rs

```diff
@@ -7631,6 +7643,34 @@ pub fn move_active_item(
     });
 }
 
+pub fn clone_active_item(
+    source: &Entity<Pane>,
+    destination: &Entity<Pane>,
+    focus_destination: bool,
+    window: &mut Window,
+    cx: &mut App,
+) {
+    if source == destination {
+        return;
+    }
+    let Some(active_item) = source.read(cx).active_item() else {
+        return;
+    };
+    destination.update(cx, |target_pane, cx| {
+        let Some(clone) = active_item.clone_on_split(None, window, cx) else {
```

Comment:
> `None`

Seems that whatever we pass there is no-op, but to be on a safe side, let's call it with the `self.database_id()` instead, as the only other `clone_on_split` invocation does.

Reply:
I think I'll need to move the function into the workspace or pane impl to do that? Not sure how to get the database id from the context. I can investigate more but if you know an easy way that would be helpful.

Reply:
That, or an extra parameter, yes.